google-cloud-service_directory/.rubocop.yml

```diff
@@ -0,0 +1,13 @@
+inherit_gem:
+  google-style: google-style.yml
+
+AllCops:
+  Exclude:
+    - "test/**/*"
+
```

Comment:
Rakefile ignored in language but not here

Reply:
The exclusion is not needed in this case. We can add it later if it does become necessary.